core/render/vi/__init__.py

```diff
@@ -163,11 +163,23 @@ def index(*args, **kwargs):
 index.exposed = True
 
 
+def get_settings():
+    fields = ["admin.name", "admin.logo", "admin.login.background", "admin.login.logo", "admin.color.primary",
+              "admin.color.secondary"]
```

Comment:
Please implement a prefix match on `field.startswith("admin.")` rather than a fixed list of fields which cannot be configured from outside.